docs/source/Mux.rst

```diff
@@ -85,15 +85,15 @@ Given these you should be able to implement any kind of parser or params
 feeder, should you require one. We favor ``yaml`` and therefor we implemented
 a ``yaml_to_mux`` plugin which can be found in
 ``avocado/plugins/yaml_to_mux.py`` and on it we also describe the way
-``Mux`` works: `yaml_to_mux plugin`_
+``Varianter`` works: `yaml_to_mux plugin`_
 
 
 Yaml_to_mux plugin
 ==================
 
 In order to get a good coverage one always needs to execute the same test
 with different parameters or in various environments. Avocado uses the
-term ``Multiplexation`` or ``Mux`` to generate multiple variants of the same
+term ``Multiplexation`` or ``Variants`` to generate multiple variants of the same
```

Comment:
We didn't define what `Multiplexation` is before... this docs update is only about keeping the names in sync with code. I'll rework the documentation from scratch with the pluggable multiplexer PR.